lib/src/main/kotlin/com/swmansion/starknet/data/types/IntWrapper.kt

```diff
@@ -0,0 +1,8 @@
+package com.swmansion.starknet.data.types
+
+import com.swmansion.starknet.data.serializers.IntWrapperSerializer
+import kotlinx.serialization.Serializable
+
+@JvmInline
+@Serializable(with = IntWrapperSerializer::class)
+value class IntWrapper(val value: Int) : HttpBatchRequestType
```

Comment:
If I remember correctly, value classes are not interoperable with underlaying types? (so they are completely new types, unlike type aliases). How will that impact users? If they call a method that returns IntWrapper type, they will need to smehow convert it to int in order to do something more with it, right?

Reply:
In order to access integer value from `StarknetInt` wrapper (`IntWrapper` before), we have to access its `value`  property. For example having such function `getBlockNumber(): HttpRequest<StarknetInt>` we will use it this way:
```kotlin
val response: StarknetInt = provider.getBlockNumber().send()
val result: Int = response.value
```